readme.md

```diff
@@ -67,6 +67,12 @@ Default: `false`
 
 Return the [main entry](https://registry.npmjs.org/ava) containing all versions.
 
+##### got
+
+Type: `object`
+
+Options to pass down to [got](https://www.npmjs.com/package/got).
```

Comment:
I would prefer not exposing this. Exposing it would mean that every breaking change in Got is a breaking change in this package.

Reply:
Makes sense. I guess it would indeed introduce a maintenance overhead. Removed.